app/src/main/java/mozilla/lockbox/presenter/BackablePresenter.kt

```diff
@@ -1,15 +1,24 @@
 package mozilla.lockbox.presenter
 
+import io.reactivex.Observable
+import io.reactivex.rxkotlin.addTo
+import mozilla.lockbox.action.RouteAction
 import mozilla.lockbox.flux.Dispatcher
 import mozilla.lockbox.flux.Presenter
 
 interface BackableView {
-    // No methods yet.
+    val backButtonClicks: Observable<Unit>
 }
 
 class BackablePresenter(
     val view: BackableView,
     val dispatcher: Dispatcher = Dispatcher.shared
 ) : Presenter() {
-    // No methods yet.
+    override fun onViewReady() {
```

Comment:
will you enforce that views inheriting from the `BackablePresenter` should call `super.onViewReady()`?

Reply:
:+1:

Reply:
Good catch. 🛁 

Right now, `BackablePresenter` is _only_ used by `BackableFragment`s — i.e. the fragments have two presenters each.